### ISSUE TITLE
ITF Barcode missing

### ISSUE THREAD

**Issue by 4Nightmares** [brarcher/loyalty-card-locker#345]
I miss ITF Barcodes
Aka interleaved-two-of-five. Would it be possible to add?

**Comment by TheLastProject** [comment]
Looking at the code, it does seem ITF is supported. Where exactly are you having issues? Scanning? Manually picking the barcode type? Something else?

**Comment by 4Nightmares** [comment]
Yes, this was for manual adding. Scanning works, but I No longer have that Card.

**Comment by brarcher** [comment]
That is, it is not possible to manually add a ITF barcode? I'm finding that on a test device it does allow manually creating a ITF barcode. Wikipedia says that it must be 14 digits, and in the activity to enter a code directly ITF is shown as an option for the input "12345678901234":

![image](https://user-images.githubusercontent.com/5264535/72781442-20f21e80-3bd6-11ea-8a4b-865c4c884a26.png)

(The titles next to the barcodes are from not yet committed https://github.com/brarcher/loyalty-card-locker/pull/333)

Are you able to give more details on how you are finding ITF barcode manual entry not working? It would help in tracking down the issue.

**Comment by 4Nightmares** [comment]
Ah ok, My code only has 6 digits and therefor also has less lines than your example. Not a big deal, I used another app on another phone to have the code generated and, after scanning, it is on loyalty-card-locker now. The code is from a German company called "engelhorn".

**Comment by TheLastProject** [comment]
Which app did you manage to generate the code with? And if you scan the card with loyalty-card-locker, does it show your 6 digit code as card id or something else?

**Comment by 4Nightmares** [comment]
I used Stocard - even if I do not like to give away personal data. And yes, lcl now shows the 6 digit code.
